### ISSUE TITLE
[A11y] Legend can be focused twice by Tab

### ISSUE THREAD

**Issue by izkizk8** [highcharts/highcharts#22526]
#### Expected behaviour
Legend should be focused just only once by Tab

#### Actual behaviour
Legend can be focused twice by Tab

#### Live demo with steps to reproduce

https://jsfiddle.net/hfs2e8xL/1/
![recording](https://github.com/user-attachments/assets/4444da4d-9a9f-4645-89ab-14dcaa4c8963)
![image](https://github.com/user-attachments/assets/fc942812-9eb6-4427-9cc3-df7ce99401e8)

#### Product version

latest

#### Affected browser(s)
all

**Comment by karolkolodziej** [comment]
Hi @izkizk8.
Thank you for describing everything in the GIF- this was helpful.

_Internal note:_
When exporting is removed, I'm not able to recreate it- [demo](https://jsfiddle.net/BlackLabel/jdqx1k46/).

**Comment by oysteinmoseng** [comment]
Aha, so to clarify: This only happens if the exporting module is imported, but the exporting button is disabled.

**Comment by izkizk8** [comment]
Hi everyone, @karolkolodziej, @oysteinmoseng ~

First of all, thanks again for your work on this! I really appreciate the quick PR that was merged.

I recently tested this again and found that the issue is still present in latest version. The behavior is the same as originally described in this issue, and the provided demos still reproduce the problem.

Would it be possible to reopen this for further investigation? Let me know if any additional information would help!

Thanks!

**Comment by oysteinmoseng** [comment]
Hi @izkizk8! The latest stable release does not yet have this fix included, but a new version is coming out shortly that will include this fix & more.

**Comment by izkizk8** [comment]
Hi @oysteinmoseng ,

Thanks for the clarification! That makes sense. I’ll keep an eye out for the release.

Just to better plan on my end, do you have an estimated timeframe for when this will be released? No worries if it's not set in stone—I just want to get a rough idea.

Appreciate your work as always! 🚀

**Comment by oysteinmoseng** [comment]
Thanks a lot! We are still ironing out some things, but within 2 weeks or so I would say.
